packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -398,6 +398,9 @@ class _GoogleMapState extends State<GoogleMap> {
   }
 
   Future<void> onPlatformViewCreated(int id) async {
+    if (!mounted) {
```

Comment:
How is this related to the PR?

Reply:
See https://github.com/flutter/flutter/issues/115283 - I can add a test for this.